scripts/image_list.rb

```diff
@@ -53,23 +67,20 @@ def deep_populate_nil_values(hash)
 }
 
 # Process the non-BOSH releases.
-values['releases'].keys.each do |release_name|
+values.releases.keys.each do |release_name|
   # Filter out the 'defaults' key as it's not a release.
   next if release_name == 'defaults'
```

Comment:
This should be `$defaults` now, for correctness. In practice it doesn't matter because the default entry will not have an `image` key, so will be skipped in the next condition.